sentry_sdk/integrations/wsgi.py

```diff
@@ -308,3 +347,22 @@ def event_processor(event, hint):
         return event
 
     return event_processor
+
+
+def _finish_long_running_transaction(current_scope, isolation_scope):
+    # type: (sentry_sdk.scope.Scope, sentry_sdk.scope.Scope) -> None
+    """
+    Make sure we don't keep transactions open for too long.
+    Triggered after MAX_TRANSACTION_DURATION_SECONDS have passed.
+    """
+    try:
+        transaction_duration = (
+            datetime.now(timezone.utc) - current_scope.transaction.start_timestamp
+        ).total_seconds()
+        if transaction_duration > MAX_TRANSACTION_DURATION_SECONDS:
```

Comment:
Do we need to check this here? Since this function can only get called when the timer fires this should always be the case

Reply:
True! (This check is some leftover code from a previous implementation=